R/translations.R

```diff
@@ -150,11 +153,13 @@ translations_direct_base <- list(
   char_length = quote(nchar),
   character_length = quote(nchar),
   concat = quote(paste0),
+  len = quote(nchar),
```

Comment:
I believe `len()` in Microsoft SQL Server does not count trailing whitespace characters. Is that correct? Do you think we should translate `len(x)` to `nchar(trimws(x, which = "right"))`?

Reply:
I think it's reasonable to _not_ remove the trailing whitespace. Similarly, SQL compares 2 strings by right-padding the shorter one to the length of the longer one. So, `'a' = 'a '` because the first is right-padded to match. We wouldn't translate that condition to `str_pad("a", 2, side = "right") == "a "`.

Reply:
Works for me, thanks.

P.S. The result of a string equality comparison like that will depend on which SQL engine you're using and which exact data types the string columns have.